tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "noEmit": true,
     "jsx": "react"
   },
-  "include": ["src"]
```

Comment:
FWIW- I think this means that website code under src/ would now be able to import code from scripts/ which we probably don't really want. 

Can we just tweak CI so that it runs `tsc` inside the `scripts` directory (which has its own [tsconfig.json](https://github.com/covid-projections/covid-projections/blob/develop/scripts/tsconfig.json) already)?

Reply:
Hm, you're right, what if we updated `"tsc"` in `packages.json` to run both `--project tsconfig.json` and `--project scripts/tsconfig.json`?

Or do we want to avoid running it on `scripts` outside of CI?

Reply:
Ah, that sounds perfect.

I suspect nobody runs `yarn tsc` manually (I didn't even realize it existed until you made this PR) and so in practice, updating it there will only affect CI.  But in general I don't think it hurts to always run against both.  I'd only hesitate if we ended up making `yarn start` slower or something, but I assume they're unrelated?

Reply:
ha ha, I run it all the time, it's my background sanity check 

I don't believe `yarn start` is related.  I'll go ahead and proceed with this change. 

... so I did, turns out the tsconfig in scripts was broken, so I fixed that.

This should work for everybody, and doesn't add much time.